src/FastMCP.ts

```diff
@@ -1402,16 +1404,13 @@ export class FastMCPSession<
           (prompt) => prompt.name === request.params.ref.name,
         );
 
-        if (!prompt) {
-          throw new UnexpectedStateError("Unknown prompt", {
-            request,
-          });
-        }
-
-        if (!prompt.complete) {
-          throw new UnexpectedStateError("Prompt does not support completion", {
-            request,
-          });
+        if (!prompt || !prompt.complete) {
+          // Return empty completion instead of throwing error
+          return {
+            completion: {
+              values: [],
+            },
+          };
```

Comment:
I don't know if this makes sense or if it is even relevant to solving your issue...

I believe there is a difference between completions, and prompt completions...

Though I haven't dug too deep in the mcp spec.